build/mbl/build.sh

```diff
@@ -522,6 +524,7 @@ Useful STAGE names:
 EOF
 }
 
+set -x
```

Comment:
That's not needed. There is a flag to enable/disable debug.

Reply:
Arg yes - that was left in for my own debug. The existing flag for enabling debug wasn't good enough for me because I wanted to debug what happened during argument parsing.

Reply:
https://github.com/ARMmbed/mbl-tools/pull/307/commits/668b3d878c87c706489a119f88be280a98a23f06